cmd/utils/flags.go

```diff
@@ -1192,11 +1291,15 @@ func setgRPC(ctx *cli.Context, cfg *node.Config) {
 		cfg.GRPCHost = "127.0.0.1"
 		if ctx.GlobalIsSet(GRPCListenAddrFlag.Name) {
 			cfg.GRPCHost = ctx.GlobalString(GRPCListenAddrFlag.Name)
+		} else if ctx.GlobalIsSet(LegacyGRPCListenAddrFlag.Name) {
```

Comment:
It is minor comment, but could you apply `getFlagName` at GRPCListenAddr and GRPCPort too?